### PR TITLE
fixed logger to allow passing of BASE_LOG_LEVEL=DISABLE

### PR DESCRIPTION
**Pull request by jakobs**

### CODE REVIEW

**Comment by doudou** [comment]
What is the semantic of BASE_LOG_LEVEL=DISABLE ?

**Comment by jakobs** [comment]
Well, to disable all logging. At least according to the definition of base/logger. Unfortunately it is broken there, but that I realised after I added this PR. Doesn't hurt though, and the functionality can be fixed in base/logger.

File: lib/utilrb/logger/root.rb [hunk 1]
```diff
@@ -41,8 +41,13 @@ class Logger
     #
     def self.Root(progname, base_level, &block)
 	begin	
-	    base_level = ENV['BASE_LOG_LEVEL'].upcase.to_sym if ENV['BASE_LOG_LEVEL']
-	    base_level = ::Logger.const_get( base_level ) if base_level.is_a? Symbol
+            if ENV['BASE_LOG_LEVEL']
+                env_level = ENV['BASE_LOG_LEVEL'].upcase.to_sym 
+                # there is currently no disabled level on the ruby side
+                unless env_level == :DISABLE
```
Comment (doudou) [inline lib/utilrb/logger/root.rb]: I would suggest to set it to FATAL, in this case, which is the closest Logger has to DISABLE

**Comment by jakobs** [comment]
Ok, updated.

**Comment by doudou** [comment]
And merged.

**Pull request merged**